content/sensu-go/5.16/api/entities.md

```diff
@@ -357,6 +391,23 @@ output               | {{< highlight json >}}
 
 The `/entities/:entity` API endpoint provides HTTP POST access to create or update the specified Sensu entity.
 
+#### EXAMPLE {#entitiesentity-post-example}
+
+In the following example, an HTTP POST request is submitted to the `/entities/:entity` API endpoint to create the entity named `sensu-centos`.
+The request includes the updated entity definition in the request body and returns a successful `HTTP 201 Created` response.
+
+{{< highlight shell >}}
+curl -X POST \
+-H "Authorization: Bearer $SENSU_TOKEN" \
+-H 'Content-Type: application/json' \
+-d '{
+PLACEHOLDER
```

Comment:
Just calling out that this is still a placeholder.

Reply:
yes -- I need someone to verify whether this call works. When I try it, I get a "405 Method Not Allowed" response.

Reply:
ah ok, I'll take a closer look then.